straxen/url_config.py

```diff
@@ -448,3 +454,36 @@ def rekey_dict(d, replace_keys='', with_keys=''):
     for old_key, new_key in zip(replace_keys, with_keys):
         new_dict[new_key] = new_dict.pop(old_key)
     return new_dict
+
+
+@URLConfig.register('objects-to-dict')
+def objects_to_dict(objects: list, key_attr=None, value_attr='value'):
```

Comment:
Worth adding an immutable dict to these options? Somthing like `immutable=False`

Reply:
done